tests/TESTS/requirements.txt

```diff
@@ -0,0 +1,37 @@
+appdirs==1.4.4
+atomicwrites==1.4.0
+attrs==20.3.0
+beautifulsoup4==4.9.3
+bleak==0.10.0
+certifi==2020.12.5
+chardet==4.0.0
+colorama==0.4.4
+fasteners==0.16
+future==0.18.2
+idna==2.10
+importlib-metadata==3.4.0
+iniconfig==1.1.1
+intelhex==2.3.0
+junit-xml==1.9
+lockfile==0.12.2
+mbed-flasher==0.10.1
+mbed-ls==1.8.4
+mbed-os-tools==1.8.4
+packaging==20.9
+pluggy==0.13.1
+prettytable==1.0.1
+py==1.10.0
+pycparser==2.20
+pyparsing==2.4.7
+pyserial==3.5
+pytest==6.2.2
+pytest-asyncio==0.14.0
+pythonnet==2.5.2
```

Comment:
why is this needed?

Reply:
The bleak client is implemented using `pythonnet` on Windows. The package provides access to the the .NET Common Language Runtime (CLR) - pip installs it w/`bleak`.

Reply:
pythonnet fails to install on linux

Reply:
Hmm... I should add [Environment Markers](https://www.python.org/dev/peps/pep-0508/#environment-markers) to the list. 

Can you install the packages directly:
`pip install pytest pytest-asyncio pyserial mbed-flasher mbed-ls`